manim/mobject/geometry/arc.py

```diff
@@ -637,10 +655,10 @@ def construct(self):
                     self.add(NumberPlane(), circle, dots)
         """
         center = line_intersection(
-            perpendicular_bisector([p1, p2]),
-            perpendicular_bisector([p2, p3]),
+            perpendicular_bisector([np.array(p1), np.array(p2)]),
+            perpendicular_bisector([np.array(p2), np.array(p3)]),
```

Comment:
Maybe `np.asarray`? Do we need to make a copy in this case?

Reply:
God idea.